### ISSUE TITLE
Read only user doesn't get the sightings when it gets an attribute

### ISSUE THREAD

**Issue by Rafiot** [MISP/MISP#5022]
POST on sightings/listSightings as a RO user says it doesn't have the right to do so.

**Comment by Rafiot** [comment]
They can see the sightings when they GET the Event, but not when they get the attribute itself (it is inconsistent with the Tag)

**Comment by iglocska** [comment]
Is this still related to the original issue with sightings/listSightings? Or is this a different endpoint that doesn't work?

**Comment by Rafiot** [comment]
No, the one that doesn't work is GET on `attributes/<ID>`. As we get the tags, I'd expect the sightings too.

**Comment by iglocska** [comment]
Mumbl, Grumbl, this was the reason I didn't want to include the tags when you've asked for those ;)

**Comment by Rafiot** [comment]
hahaha
